reference/5.1/Microsoft.PowerShell.Core/About/about_Command_Precedence.md

```diff
@@ -145,15 +145,27 @@ For cmdlets, you can use the module-qualified name. For executables, you can
 include the file extension. For example, to run the executable version of
 `where` use `where.exe`.
 
-### Using module-qualified names
+### Use module-qualified names
 
 Using the module-qualified name of a cmdlet allows you to run commands hidden
 by an item with the same name. For example, you can run the `Get-Date` cmdlet
-by qualifying it with its module name **Microsoft.PowerShell.Utility**.
+by qualifying it with its module name **Microsoft.PowerShell.Utility** or full
+path. When you use module-qualified names, the module is automatically imported
```

Comment:
```suggestion
path. When you use module-qualified names, the module is automatically imported if `$psModuleAutoLoadingPreference` is set to `all`
```

Reply:
we could say, based of `$psModuleAutoLoadingPreference` and link to here:
https://github.com/MicrosoftDocs/PowerShell-Docs/blob/6feaff6bce4a5135b83030dbbc3a4393ab3316b0/reference/5.1/Microsoft.PowerShell.Core/About/about_Preference_Variables.md?plain=1#L1092

Reply:
Added